flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java

```diff
@@ -285,6 +288,16 @@ public boolean isExternalizedCheckpointsEnabled() {
 		return externalizedCheckpointCleanup != null;
 	}
 
+	@PublicEvolving
+	public boolean isPreferCheckpointForRecovery() {
+		return perferCheckpointForRecovery;
```

Comment:
There are still a few typos if you search for "perfer"

Reply:
Sorry, will fix them.